src/components/AdministrationPage/AdministrationPage.tsx

```diff
@@ -23,6 +30,28 @@ export const AdministrationPage = () => {
     paddingRight: '8px',
     paddingBottom: '5px',
   };
+  const [secondsUntilReload, setSecondsUntilReload] = useState<number>(REFRESH_INTERVAL_SECONDS);
+  const dataContainer = useDataContainer(
+    useCallback(({ requestConfig }: IService) => buildService.getBuildCount(requestConfig), [])
+  );
+  const dataContainerRefresh = dataContainer.refresh;
+  // TODO: Create a better solution than disabling the next line
+  useEffect(() => dataContainer.refresh({}), []); // eslint-disable-line react-hooks/exhaustive-deps
+
+  const refreshBuildCounts = useCallback(() => {
+    dataContainerRefresh({});
```

Comment:
is passing `{}` necessary?

Reply:
I believe it is, not passing it causes an error 
![image](https://user-images.githubusercontent.com/37015743/176138090-8c405202-d38d-4785-9a93-06c80e4639b6.png)